src/App.tsx

```diff
@@ -71,7 +93,8 @@ const settingsNav = {
     {
       name: "Users",
       href: "/settings/users",
-      icon: HiUser
+      icon: HiUser,
+      resouceName: RESOURCES["SETTINGS.USERS"]
```

Comment:
Why isn't this a variable like the others ?

Reply:
I have grouped all resources under `RESOURCES` constant so that while using we can just import `RESOURCES` instead of importing each of them individually.